week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -32,23 +32,35 @@ var parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
-
+  return house.currentOwner.firstName + " " + house.currentOwner.lastName;
 }
+console.log(getOwnerFullName(kinningParkHouse));
 
 // returns an array of the owners' email addresses of the two houses
 function getEmailAddresses(house1, house2) {
-
+  return house1.currentOwner.email + "" + house2.currentOwner.email;
 }
+console.log();
 
 // returns the address for the cheapest house out of the two
 function getCheapestAddress(house1, house2) {
-
+  if (house1.price > house2.price) {
+    return house2.address;
+  }

```

Comment:
There are two possible "execution paths" here. If house1 is cheaper you'd return the address of house1 and if house2 is cheaper you'd return the address of house2. Are you sure you've covered both paths?